pep-0603.rst

```diff
@@ -76,8 +75,8 @@ Construction
 
 ``frozenmap`` implements a ``dict``-like construction API:
 
-* ``frozenmap()`` creates a new empty mapping (TODO: clarify...immutable after the first
-  write of data???);
+* ``frozenmap()`` creates a new empty mapping (which can be populated with
+  data using the ``set`` method.);
```

Comment:
"which can be populated with data using the ``set`` method" -- this is slightly inaccurate as we aren't actually populating *that particular* instance.  With set() we can create new frozenmaps out of an empty one, but I'm not sure it's useful to note.

This is similar to `dict()` creating an empty dictionary kind of thing -- we need to document it, but maybe it isn't that useful.